tests/e2e/fixtures/statsFixtures.js

```diff
@@ -263,6 +268,64 @@ const seasonTwoMatches = [
   },
 ];
 
+const seasonFourMatches = [
+  ////////////
+  // Week 1 //
+  ////////////
+  // Player 1 week 1
+  {
+    player1: 'player1',
+    player2: 'player2',
+    winner: 'player2',
+    startTime: dayjs('2023-07-20', 'yyyy-mm-dd').valueOf(),
+    endTime: dayjs('2023-07-20', 'yyyy-mm-dd').valueOf(),
+  },
+  {
+    player1: 'player1',
+    player2: 'player3',
+    winner: 'player1',
+    startTime: dayjs('2023-08-20', 'yyyy-mm-dd').valueOf(),
+    endTime: dayjs('2023-08-20', 'yyyy-mm-dd').valueOf(),
+  },
+  {
+    player1: 'player1',
+    player2: 'player4',
+    winner: 'player4',
+    startTime: dayjs('2023-08-25', 'yyyy-mm-dd').valueOf(),
+    endTime: dayjs('2023-08-25', 'yyyy-mm-dd').valueOf(),
+  },
+  // Player 2 week 1
+  {
+    player1: 'player2',
+    player2: 'player3',
+    winner: 'player3',
+    startTime: dayjs('2023-09-15', 'yyyy-mm-dd').valueOf(),
+    endTime: dayjs('2023-09-15', 'yyyy-mm-dd').valueOf(),
+  },
+  {
+    player1: 'player2',
+    player2: 'player4',
+    winner: 'player2',
+    startTime: dayjs('2023-09-20', 'yyyy-mm-dd').valueOf(),
+    endTime: dayjs('2023-09-20', 'yyyy-mm-dd').valueOf(),
+  },
+  // Player 3 week 1
+  {
+    player1: 'player3',
+    player2: 'player4',
+    winner: 'player3',
+    startTime: dayjs('2023-10-09', 'yyyy-mm-dd').valueOf(),
+    endTime: dayjs('2023- 10-09', 'yyyy-mm-dd').valueOf(),
+  },
+  {
+    player1: 'player3',
+    player2: 'player5',
+    winner: 'player3',
+    startTime: dayjs('2023-10-15', 'yyyy-mm-dd').valueOf(),
+    endTime: dayjs('2023-10-15', 'yyyy-mm-dd').valueOf(),
+  },
```

Comment:
Same idea as above, once we make it so the current season is calculated relative no now instead of hard coded dates, these will need to update to be relative to now